api/helpers/game-states/moves/seven-scuttle/execute.js

```diff
@@ -0,0 +1,55 @@
+const GamePhase = require('../../../../../utils/GamePhase.json');
+
+module.exports = {
+  friendlyName: 'Seven_Scuttle',
+
+  description: 'Returns new GameState resulting from requested scuttle move',
+
+  inputs: {
+    currentState: {
+      type: 'ref',
+      description: 'The latest GameState before the requesting player scuttles',
+      required: true,
+    },
+    /**
+     * @param { Object } requestedMove - Object describing the request to scuttle from the top of the deck
+     * @param { String } requestMove.cardId - Card Played 
+     * @param { String } requestMove.targetId - Card Targeted
+     * @param { MoveType.SEVEN_SCUTTLE } requestedMove.moveType - Specifies that this a sevenScuttle
+     */
+    requestedMove: {
+      type: 'ref',
+      description: 'The move being requested. Specifies moveType, card played and targeted',
+    },
+    playedBy: {
+      type: 'number',
+      description: 'Player number of player requesting move.',
+    },
+  },
+  sync: true,
+  fn: ({ currentState, requestedMove, playedBy }, exits) => {
+    const { cardId } = requestedMove;
+    let result = _.cloneDeep(currentState);
+
+    const player = playedBy ? result.p1 : result.p0;
```

Comment:
We don't need player, but we do need opponent. This is because the cards in the player hand and points don't change for this move, but the cards from the opponent do since we move the targetCard from their points to the scrap
```suggestion
    const opponent = playedBy ? result.p0 : result.p1;
```